tool/tctl/common/tctl.go

```diff
@@ -288,6 +288,10 @@ func applyConfig(ccf *GlobalCLIFlags, cfg *service.Config) (*authclient.Config,
 				return nil, trace.Wrap(err, fmt.Sprintf("Could not load Teleport host UUID file at %s. "+
 					"Please make sure that Teleport is up and running prior to using tctl.",
 					filepath.Join(cfg.DataDir, utils.HostUUIDFile)))
+			} else if errors.Is(err, fs.ErrPermission) {
```

Comment:
A couple comments:

1. Why the inconsistency? In `tbot` we are logging an enhanced error message, but returning the original error unmodified. In `tctl` we are wrapping the original error with a better message. Can we be consistent and take the same approach in both places?
2. You shouldn't need to use `fmt.Sprintf` here. `trace.Wrap` already supports this: `trace.Wrap(err, "this is a message: %v", x)`

Reply:
thanks @zmb3 I've made these consistent approaches and removed the unnecessary fmt.Sprintf.  Please check when you can.